analytics/js_tracking_api.rst

```diff
@@ -144,41 +148,42 @@ Ecommerce tracking
 
 Adding Ecommerce item
 `````````````````````
-To add ecommerce item (for example to track changes in users cart using ``trackEcommerceCartUpdate``) user ``addEcommerceItem`` function::
+To add ecommerce item (for example to track changes in users cart using ``trackEcommerceCartUpdate``) use ``addEcommerceItem`` function::
 
     _paq.push(["addEcommerceItem", productSKU, productName, productCategory, productPrice, productQuantity]);
 
+.. note::
+
+    This function does not track any data to Tracker. This is helper function for :ref:`trackEcommerceOrder <Tracking Ecommerce order>`   or :ref:`trackEcommerceCartUpdate <Updating cart>` function to populate items to send.
```

Comment:
Proposition:

This function does not send any data to the analytics (tracker, server?). It only prepares Ecommerce cart/order state to be send with :ref:`trackEcommerceOrder <Tracking Ecommerce order>` or :ref:`trackEcommerceCartUpdate <Updating cart>`.